test/data/generate_data.py

```diff
@@ -76,6 +77,14 @@ async def main(ped_path=default_ped_location, project='greek-myth'):
         await papi.create_project_async(
             name=project, dataset=project, create_test_project=False
         )
+        default_user = os.getenv('SM_LOCALONLY_DEFAULTUSER')
```

Comment:
Some defensiveness here would be valuable - I don't usually have this environment variable set in my bash environments